src/docs/MLOS/MLOSUserGuide.adoc

```diff
@@ -791,6 +802,94 @@ pip install py4j# <2>
 <1> _Connexion_ allows you to write an OpenAPI specification, then maps the endpoints to your Python functions.
 <2> _py4j_ enables Python programs running in a Python interpreter to dynamically access Java objects in a Java Virtual Machine.
 
+=== Data Drift Detection (DDD)
+The data evolves over time and can therefore cause degradations affecting the
+intrinsic characteristics and behavior of the learning model.
+Data drift is one of the main reasons why the accuracy of the model degrades over time.
+Therefore, it is important that the model is able to adapt to these changes.
+In machine learning, data drift is the modification of the model input data (relative to the
+training data) that leads to a degradation of the model performance.
+Monitoring data drifts allows to detect
+these model performance problems and try to adapt to them as in the figure below. It is in this context that we were interested in the
+integration of data drift detection in the Machine Learning as a Service module of
```

Comment:
It is in this context that we were interested => To deal with this problem, we integrated a data drift detection mechanism in the Machine Learning as a Service ..
Machine Learning as a Service => this should be updated according to the service naming that we will use